oada-core/http-handler/oada-srvc-http-handler/websockets.ts

```diff
@@ -0,0 +1,434 @@
+                            resourceId,
+                            request.headers['x-oada-rev']
+                        )
+                        const rev = await resources.getResource(
+                            resourceId,
+                            '_rev'
+                        )
+                        // If the requested rev is behind by revLimit, simply
+                        // re-GET the entire resource
+                        trace(
+                            'REVS:',
+                            resourceId,
+                            rev,
+                            request.headers['x-oada-rev']
+                        )
+                        if (
+                            parseInt(rev) -
+                                parseInt(request.headers['x-oada-rev']) >=
+                            revLimit
+                        ) {
+                            trace(
+                                'REV WAY OUT OF DATE',
+                                resourceId,
+                                rev,
+                                request.headers['x-oada-rev']
+                            )
+                            const resource = await resources.getResource(
+                                resourceId
+                            )
+                            sendResponse({
+                                requestId: msg.requestId,
+                                resourceId,
+                                resource,
+                                status: 'success'
+                            })
+                        } else {
+                            // First, declare success.
+                            sendResponse({
+                                requestId: msg.requestId,
+                                status: 'success'
+                            })
+                            trace(
+                                'REV NOT TOO OLD...',
+                                resourceId,
+                                rev,
+                                request.headers['x-oada-rev']
+                            )
+                            // Next, feed changes to client
+                            const newChanges = await changes.getChangesSinceRev(
+                                resourceId,
+                                request.headers['x-oada-rev']
+                            )
+                            newChanges.forEach(change =>
+                                sendChange({
+                                    requestId: [msg.requestId],
+                                    resourceId,
+                                    path_leftover,
+                                    change
+                                })
+                            )
+                        }
+                    } else {
+                        sendResponse({
+                            requestId: msg.requestId,
+                            status: 'success'
```

Comment:
I know this is not part of your changes, but can we use an integer status code here (e.g., 2xx)?

Reply:
Fine with me. I was just trying to make it backwards compatible.